### DISCUSSION TITLE
Linux Version

### DISCUSSION THREAD

**Discussion by z3nitsu** [OpenLoco/OpenLoco#1500] [General]
Will there be a Linux version is future ?

**Comment by AaronVanGeffen** [comment]
There already is. See our [readme](https://github.com/OpenLoco/OpenLoco#linux--macos-1) and [wiki](https://github.com/OpenLoco/OpenLoco/wiki/Building-on-Ubuntu). What distro are you using?

**Reply by z3nitsu** [reply]
I'm using POP os, trying to compile the game but getting an error

**Reply by AaronVanGeffen** [reply]
It would help to include the error message.

**Reply by z3nitsu** [reply]
I included the error message below

**Comment by z3nitsu** [comment]
CMake Error at /usr/share/cmake-3.22/Modules/CMakeTestCXXCompiler.cmake:62 (message):
  The C++ compiler

    "/usr/bin/c++"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /home/z3nitsu/Desktop/OpenLoco/build/CMakeFiles/CMakeTmp
    
    Run Build Command(s):/usr/bin/gmake -f Makefile cmTC_5828d/fast && /usr/bin/gmake  -f CMakeFiles/cmTC_5828d.dir/build.make CMakeFiles/cmTC_5828d.dir/build
    gmake[1]: Entering directory '/home/z3nitsu/Desktop/OpenLoco/build/CMakeFiles/CMakeTmp'
    Building CXX object CMakeFiles/cmTC_5828d.dir/testCXXCompiler.cxx.o
    /usr/bin/c++   -m32  -o CMakeFiles/cmTC_5828d.dir/testCXXCompiler.cxx.o -c /home/z3nitsu/Desktop/OpenLoco/build/CMakeFiles/CMakeTmp/testCXXCompiler.cxx
    Linking CXX executable cmTC_5828d
    /usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_5828d.dir/link.txt --verbose=1
    /usr/bin/c++ -m32  CMakeFiles/cmTC_5828d.dir/testCXXCompiler.cxx.o -o cmTC_5828d 
    /usr/bin/ld: cannot find Scrt1.o: No such file or directory
    /usr/bin/ld: cannot find crti.o: No such file or directory
    /usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/11/libstdc++.so when searching for -lstdc++
    /usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/11/libstdc++.a when searching for -lstdc++
    /usr/bin/ld: cannot find -lstdc++: No such file or directory
    /usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/11/libstdc++.so when searching for -lstdc++
    /usr/bin/ld: skipping incompatible /usr/lib/x86_64-linux-gnu/libm.so when searching for -lm
    /usr/bin/ld: skipping incompatible /usr/lib/x86_64-linux-gnu/libm.a when searching for -lm
    /usr/bin/ld: cannot find -lm: No such file or directory
    /usr/bin/ld: skipping incompatible /usr/lib/x86_64-linux-gnu/libm.so when searching for -lm
    /usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/11/libgcc.a when searching for -lgcc
    /usr/bin/ld: cannot find -lgcc: No such file or directory
    collect2: error: ld returned 1 exit status
    gmake[1]: *** [CMakeFiles/cmTC_5828d.dir/build.make:99: cmTC_5828d] Error 1
    gmake[1]: Leaving directory '/home/z3nitsu/Desktop/OpenLoco/build/CMakeFiles/CMakeTmp'
    gmake: *** [Makefile:127: cmTC_5828d/fast] Error 2
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:21 (project)

**Reply by AaronVanGeffen** [reply]
You appear to be linking with 64-bit libraries. We cannot support 64-bit builds yet. Did you invoke cmake with `-m32`?

**Reply by z3nitsu** [reply]
yes, I did

**Reply by z3nitsu** [reply]
I was missing one library and now it working
